### PR TITLE
Add a table of common jars that package native code and status for Ar…

### PR DESCRIPTION
**Pull request by geoffreyblake**
*Description of changes:*

Create a best-effort table or Jars that customers commonly need to use that contain native code and the status of their Arm support.  

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CommonNativeJarsTable.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Table of commonly used Jars that package native artifacts
+
+Org  | jar  | Builds on Arm | Arm Artifact available
+-----|------|---------------|--------------------
```
Comment (AGSaidi) [inline CommonNativeJarsTable.md:4]: how about links to the github repos in the jar column and links to the artifacts in the arm artifact available.

File: README.md [hunk 2]
```diff
@@ -220,7 +220,13 @@ $ find . -name "*.so" | xargs file
 For each x86-64 ELF file, check there is a corresponding aarch64 ELF file
 in the binaries. With some common packages (e.g. commons-crypto) we've seen that
 even though a JAR can be built supporting Arm manually, artifact repositories such as
-Maven don't have updated versions. Feel free to open an issue in this GitHub repo for advice.
+Maven don't have updated versions. To see if a certain artifact version may have Arm support,
+consult our [Common JARs with native code Table](CommonNativeJarsTable.md)
+which is kept up-to-date on best effort and through PRs.
```
Comment (AGSaidi) [inline README.md:225]: given it's a github repo it feels like this is redundant.

File: CommonNativeJarsTable.md (hunk 1 above)
Comment (geoffreyblake) [reply CommonNativeJarsTable.md:4]: Done.

File: README.md (hunk 2 above)
Comment (geoffreyblake) [reply README.md]: Agree, fixed.

**Pull request merged**